PhysObjectExtractor/src/MuonAnalyzer.cc

```diff
@@ -84,14 +98,45 @@ MuonAnalyzer::MuonAnalyzer(const edm::ParameterSet& iConfig)
 	mtree = fs->make<TTree>("Events", "Events");
 
 	mtree->Branch("numbermuon",&nummuon);
+	mtree->GetBranch("numbermuon")->SetTitle("number of muons");
 	mtree->Branch("muon_e",&muon_e);
+	mtree->GetBranch("muon_e")->SetTitle("muon energy");
 	mtree->Branch("muon_pt",&muon_pt);
+	mtree->GetBranch("muon_pt")->SetTitle("muon transverse momentum");
  	mtree->Branch("muon_px",&muon_px);
+ 	mtree->GetBranch("muon_px")->SetTitle("muon momentum x-component");
  	mtree->Branch("muon_py",&muon_py);
+ 	mtree->GetBranch("muon_py")->SetTitle("muon momentum y-component");
 	mtree->Branch("muon_pz",&muon_pz);
+	mtree->GetBranch("muon_pz")->SetTitle("muon momentum z-component");
  	mtree->Branch("muon_eta",&muon_eta);
+ 	mtree->GetBranch("muon_eta")->SetTitle("muon pseudorapidity");
 	mtree->Branch("muon_phi",&muon_phi);
+	mtree->GetBranch("muon_phi")->SetTitle("muon polar angle");
 	mtree->Branch("muon_ch",&muon_ch);
+	mtree->GetBranch("muon_ch")->SetTitle("muon charge");
+	mtree->Branch("muon_mass",&muon_mass);
+	mtree->GetBranch("muon_mass")->SetTitle("muon mass");
+	mtree->Branch("muon_pfreliso03all",&muon_pfreliso03all);
+	mtree->GetBranch("muon_pfreliso03all")->SetTitle("muon pfreliso03all");
```

Comment:
Maybe a more descriptive title like "muon particle flow relative isolation cone 03" and similar for the 04 cone.  @manda07A I will explain later.